File: Programmers/src/Javascript/Lv0/치킨쿠폰.js
function solution(chicken) {    
    let answer = 0;
    
    while(true) {
        answer += (parseInt)(chicken/10);
        chicken = (chicken % 10) + (parseInt)(chicken/10);
        
        if((parseInt)(chicken/10) === 0)
            break;
    }
    
    return answer;
}

File: Programmers/src/Java/Lv0/치킨 쿠폰.java
class Solution {
    public int solution(int chicken) {
        int answer = 0;
        
        while(chicken >= 10) {
            int newChicken = chicken / 10;
            int rest = chicken % 10;
            chicken = newChicken + rest;
            
            answer += newChicken;
        }
        
        return answer;
    }
}